src/device-manager/cocoa/NLWdmClientFlushUpdateError.h

```diff
@@ -0,0 +1,41 @@
+/*
+ *
+ *    Copyright (c) 2020 Google, LLC.
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      Translation of WdmClientFlushUpdate errors into native Objective C types.
+ *
+ */
+
+#import <Foundation/Foundation.h>
+#import "NLWeaveDeviceManagerTypes.h"
+#import "NLWeaveErrorCodes.h"
+#import "NLWeaveError.h"
+#import "NLGenericTraitUpdatableDataSink.h"
+
+@interface NLWdmClientFlushUpdateError : NLWeaveError
```

Comment:
It's not super clear to me what the difference is between `NLWdmClientFlushUpdateError` and `NLWdmClientFlushUpdateDeviceStatusError`, might be a Weave convention I'm not familiar with, but if not, could be useful to add a documentation comment clarifying, maybe?

Reply:
NLWdmClientFlushUpdateError is mobile client side error(local error), NLWdmClientFlushUpdateDeviceStatusError is remote device error

Reply:
Yep, update the description in h file